README.md

```diff
@@ -52,7 +53,34 @@ This can be caused from any of the following: Wrong location, not enough disk sp
 The customer gets an email from the panel to setup their account (incl. password) if they didn't have an account before. Otherwise they should be able to use their existing credentials.
 
 ## My game requires multiple ports allocated.
-Currently, this isn't possible with this module but is planned.
+Configure the "**Additional Ports**" option in the module settings.
+It expects a valid JSON string consisting of the parameter name or NONE, and a number representing a port as a numeric offset from the first available allocation.
+E.g: If you enter `{"NONE":1, "NONE":2, "NONE":4}` and the first available port happens to be 25565, you'll get as additional allocations: 
```

Comment:
Could we perhaps use `{1:"NONE",2:"NONE",4:"NONE"}` instead (keys and values flipped) so that we could avoid having to deal with "invalid" JSON in a hacky way? (replacing NONE keys with unique values as the module currently does)

Reply:
Good point, not sure why I didn't see this sooner - Done and done!

Reply:
> Could we perhaps use `{1:"NONE",2:"NONE",4:"NONE"}` instead (keys and values flipped) so that we could avoid having to deal with "invalid" JSON in a hacky way? (replacing NONE keys with unique values as the module currently does)

That's not a valid JSON.
Keys must be strings in json, so it should be like this `{"1":"NONE","2":"NONE","4":"NONE"}`